test/commands/devtool.test.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+// const { test: oclifTest } = require("@oclif/test");
+const {stdout} = require("stdout-stderr");
+const assert = require("assert");
+const promisify = require('util').promisify;
+const sleep = promisify(setTimeout);
+const fetch = require('node-fetch');
+
+describe("devtool command", function() {
+    it("devtool starts and serves html", async function() {
+        // set up server
+        const DevToolCommand = require("../../src/commands/asset-compute/devtool");
```

Comment:
This require() should move to the top (and all others below). Invoking require multiple times makes no difference, the result is cached.